packages/common/components/content/blocks/query-hero.marko

```diff
@@ -7,12 +7,25 @@ $ const { site } = out.global;
 $ const block = 'content-query-hero';
 $ const params = {
   limit: 5,
+  cardImageModifiers: ['fluid','21by9'],
+  cardImageHeight: 270,
+  cardImageWidth: 630,
+  cardImageFit: 'crop',
+  cardImageCrop: 'focalpoint',
+  cardImagePosition: 'top',
+  listImageModifier: [],
+  listImageHeight: 75,
+  listImageWidth: 75,
+  listImageFit: 'crop',
+  listImageCrop: 'focalpoint',
+  listImagePosition: 'right',
```

Comment:
Make this into two `object` type arguments for `card-image-options` and `list-image-options`. The types and default values should be specified in the relevant `marko.js` file.

Since you're not passing params around, just destructure these from `input` to clean up the code a bit:
```
const {
  width: listImageWidth,
  // ...
} = input.listImageOptions;

const {
  width: cardImageWidth,
  // ...
} = input.cardImageOptions;

$ const listImageOptions = { w: listImageWidth, ... };
$ const cardImageOptions = { w: cardImageWidth, ... };
```